daemon/loadbalancer.go

```diff
@@ -903,7 +903,9 @@ func restoreServices() {
 		// Restore the service cache to guarantee backend ordering
 		// across restarts
 		if err := lbmap.RestoreService(svc, v2Exists); err != nil {
-			log.WithError(err).Warning("Unable to restore service in cache")
+			scopedLog.WithError(err).Warning("Unable to restore service in cache")
+			failed++
+			continue
```

Comment:
Shouldn't we restore the legacy maps in case of an error? :/

Reply:
Unfortunately not. The cache is required for the legacy service endpoint hole handling.